view/frontend/templates/multishipping/multishipping.phtml

```diff
@@ -0,0 +1,150 @@
+<?php
+
+use Midtrans\Snap\Block\Payment\Multishipping;
+
+/**
+ *  Midtrans multishipping payment page
+ *
+ * @var Multishipping $block
+ */
+?>
+<h2 class="price-label">Grand Total: <b><?php echo $block->getGrossAmount()?></b></h2>
+<hr>
+
+<form action="#" id="multishipping-billing-form" class="form multicheckout billing">
+    <div class="block block-billing">
+        <div class="block-content">
+            <div class="box box-billing-address">
+                <fieldset class="fieldset">
+                    <legend class="legend box-title">
+                        <span><?= $block->escapeHtml(__('Billing Address')); ?></span>
+                    </legend><br>
+                    <div class="box-content">
+                        <?php foreach ($block->getBillingAddress() as $key => $item) {
+                            echo $item . "<br/>";
```

Comment:
recommended to use `$block->escapeHtml`  when printing variable like `$item` for security